Source/Core/Core/PowerPC/TableGen.cpp

```diff
@@ -0,0 +1,459 @@
+#include <bitset>
```

Comment:
Missing the top-level comment.

Reply:
You likely meant to put 2018 there, not 2008.

Reply:
True, though truth be told, I just copypasted the comment from PPCTables.cpp …